app.js

```diff
@@ -49,11 +50,11 @@ const applyProxy = (app, config) => {
   }
 };
 
-const health = healthcheck.configure({
+let healthConfig = {
   checks: {}
-});
-app.get('/', health);
```

Comment:
with the current change do we still redirect to health when we call `/` ?
minor and ingnorable: Shall we add a test for it?

Reply:
good catch Mario, it wasn't working. Added explicit config for `/` and a unit test for it to assert it shows up health status.